servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -645,18 +645,13 @@ public FileBlockInfo getFileBlockInfo(long fileId, int fileBlockIndex)
   private FileBlockInfo generateFileBlockInfo(InodeFile file, BlockInfo blockInfo) {
```

Comment:
Are there other methods in this class that return BlockInfo, that can be removed?